react/features/app/getRouteToRender.js

```diff
@@ -0,0 +1,22 @@
+/* @flow */
+
+import { isRoomValid } from '../base/conference';
+import { RouteRegistry } from '../base/react';
+import { toState } from '../base/redux';
+import { Conference } from '../conference';
+import { WelcomePage } from '../welcome';
+
+/**
+ * Determines which route is to be rendered in order to depict a specific Redux
+ * store.
+ *
+ * @param {(Object|Function)} stateOrGetState - Redux state or Regux getState()
+ * method.
+ * @returns {Route}
+ */
+export function _getRouteToRender(stateOrGetState: Object | Function) {
```

Comment:
Why move this function to its own file? I'm not against the move but the reason isn't apparent to me immediately. Was it because it a .web.js file was importing a .native.js file?

Reply:
Exactly, web was importing native and we cannot put there some native stuff.